platform/rpc/nats/bus_test.go

```diff
@@ -0,0 +1,190 @@
+package nats
```

Comment:
Yeah don't feel strongly either. If `gnatsd` was more trouble to run than `go get ... ; gnatsd &` then I'd be more inclined to make the tests opt-in.